contracts/oracles/SAvaxOracle.sol

```diff
@@ -47,6 +46,6 @@ contract SAvaxOracle is IPriceOracle {
     }
 
     function decimals() external view override returns (uint256) {
-        return sAVAX.decimals();
+        return WAVAXOracle.decimals();
```

Comment:
We can just hardcode to 8 here right?

Reply:
Yeah I could but I think it's better to reference the underlying so we know why it's hardcoded to 8